common/index.ts

```diff
@@ -0,0 +1,10 @@
+export * from "./utils/jwt";
+export * from "./utils/storage";
+export * from "./utils/validation";
+export * from "./types/auth.type";
+export * from "./types/post.type";
+export * from "./types/posts";
+export * from "./types/error.type";
+export * from "./helpers/fetch";
+export * from "./helpers/mutate";
+export * from "./helpers/request";
```

Comment:
It might be a good idea to avoid this barrel import to keep the context of the directories when importing (/types, /helpers, /utils, etc)

Reply:
Maybe do the barrel export inside of each subdirectory like the ones I mentioned